annotator/misc/fromnpm/fromnpm.go

```diff
@@ -247,7 +247,7 @@ func MapNPMProjectRootsToPackages(packages []*extractor.Package) map[string][]*e
 
 func npmProjectRootDirectory(path string) string {
 	// Only consider packages from root/node_modules/../package.json.
-	if !(filepath.Base(path) == "package.json" && strings.Contains(path, nodeModulesDirectory)) {
+	if filepath.Base(path) != "package.json" || !strings.Contains(path, nodeModulesDirectory) {
```

Comment:
This is an example where I think it's less clear.

"Not a package.json under nodeModulesDirectory" seems clearer to me than
"Not package.json or not under nodeModulesDirectory"